src/index.njk

```diff
@@ -8,6 +8,22 @@ layout: "layouts/base.njk"
     <h1 class="rvt-ts-46 rvt-text-bold rvt-lh-title">{{ title }}</h1>
     <p class="rvt-ts-20">{{ description }}</p>
     <hr>
+    <ul class="rvt-icons-demo rvt-p-top-xxl">
```

Comment:
@basham I'm not sure we'll ultimately need to duplicate the icon examples here in the example page. I'd lean toward showing one implementation or the other (svg symbols, or CSS) for the icons demos/examples.

It's starting to feel like this demo page might need more actual documentation opposed to just the big list of icons. 🤔  Especially now that you've created this script that could be used to create custom sets of icons/css classes.

I wonder if you could take a crack at mapping out some documentation on the different implementations in the README.md, including at least a draft of some docs on how to implement this new CSS version? Then we can go from there on how we want to update the actual gh-pages demo page.

Reply:
Yes, that's a good strategy. I'm going to also abstract out the sprite generator into its own, similar function. It may result in no longer needing gulp.

Reply:
@basham Oh, awesome! I would LOVE to get rid of Gulp if possible. It's something I'm on a mission to do in other Rivet-related repos, so by all means that would definitely be a welcome change. 🙌🏻